Exercise_3.py

```diff
@@ -0,0 +1,54 @@
+#Time Complexity : O(M*N), N being the length of the array and M being the width. 
+#Space Complexity : O(M*N), N being the length of the array and M being the width. 
```

Comment:
SC is O(1) because no extra space is used here.

Reply:
That is true.

Reply:
Adjusted.